name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/messages_de.properties

```diff
@@ -2095,6 +2099,8 @@ PrefDescriptionAlphaVantage = Alpha Vantage Inc. ist ein Anbieter von kostenlose
 
 PrefDescriptionDivvyDiary = \u00DCbertrage deinen Aktienbestand automatisiert von Portfolio Performance in dein pers\u00F6nliches Depot bei DivvyDiary. Dadurch kannst du deine zuk\u00FCnftigen Dividendenzahlungen auch ganz einfach von dem Smartphone aus beobachten.\n\nBefolge daf\u00FCr einfach die folgenden Schritte:\n1. Erstelle einen Account bei DivvyDiary\n2. Generiere unter "Einstellungen" einen API-Schl\u00FCssel\n3. Hinterlege den API-Schl\u00FCssel in den Einstellungen von Portfolio Performance
 
+PrefDescriptionMyDividends24 = Verbinde dein Aktienportfolio von Portfolio Performance problemlos mit deinem pers\u00F6nlichen Konto auf myDividends24. Dies erm\u02C6glicht es dir, zuk�nftige Dividendenzahlungen bequem von deinem Smartphone aus im Blick zu behalten.\n\nBefolge einfach diese Schritte:\n1. Erstelle einen Account bei myDividends24\n2. Generiere unter "Einstellungen" oder unter dem Bereich "Portfolio/Import Portfolio Performance" einen API-Schl\u00FCssel\n3. Hinterlege den API-Schl\u00FCssel in den Einstellungen von Portfolio Performance.\n\nSo hast du alle Dividendenzahlungen in deinem myDividends24-Depot stets im \u2039berblick.
```

Comment:
Typo e.g. "zuk¸nftige" -> Unicode -> "zuk\u00FCnftigen"
Please use [ResourceBundle Editor](https://github.com/buchen/portfolio/blob/master/CONTRIBUTING.md#install-eclipse-plugins) when creating, editing and translating labels.